### ISSUE TITLE
no connection to certain access point devices possible 

### ISSUE THREAD

**Issue by momoxxl** [GrapheneOS/os-issue-tracker#2614]
I have two access points in use. The hardware is different (FritzBox and T-Com device) but the wlan configuration is identical, that means the SSIDs in the 2.4 Ghz and 5 Ghz band are identical, as well as the passwords. Initially, the wlan configuration of my Pixel 8 had the FirtzBox in range. When I am in the range of the T-Com device, it is not used.  
A test showed that the Pixel 8 cannot establish a wifi connection to the T-Com device (Speedport Smart 2) at all. My Pixel 4a can, as can my Linux laptops.  It is indicated that the password would be incorrect. But this is not the case.

The access point only supports WPA2. The 2.4 Ghz band uses 802.11b + g +n , the 5 Ghz band uses 802.11ac.

**Comment by thestinger** [comment]
Have you tried forgetting the network and doing it again?

**Comment by momoxxl** [comment]
Yes, I have also tried that. I was very surprised that it didn't work either. I also changed the SSID to make sure I was connecting to the correct access point.

Am 27. Oktober 2023 18:21:04 MESZ schrieb Daniel Micay ***@***.***>:

**Comment by momoxxl** [comment]
Yes,  I tried this already.

Am 27. Oktober 2023 18:21:04 MESZ schrieb Daniel Micay ***@***.***>:

**Comment by momoxxl** [comment]
I did some research on the internet about the hardware of the access point. It seems that (some) Pixel devices have problem with this AP. I found some reviews saying that a Pixel 3 has the same problem as I have. So I close this issue.
